### ISSUE TITLE
Need to modify regex for email checking

### ISSUE THREAD

**Issue by garima-agal** [ReactiveRaven/jqBootstrapValidation#176]
Email validation restricts the no of characters after dot(.) to be maximum of four only.

Current regex:- [a-zA-Z0-9.!#$%&\u2019*+/=?^_`{|}~-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,4}

But there are email addresses that have more than 4 characters.
For example:- info@abc.global
abc@abc.career

So, validation gets fail. Need to increase the character size after dot.

**Comment by guipacheco2** [comment]
You can override the default function to workaround.

```javascript
$.jqBootstrapValidation('override', {
    builtInValidators: {
        validemail: {
            name: 'Validemail',
            type: 'regex',
            regex: '[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\\\.[A-Za-z]{2,6}',
            message: 'Not a valid email address'
        }
    }
});
```

**Comment by garima-agal** [comment]
Yes, I did that but it should be fixed in the core rather than overriding....

**Comment by anchev** [comment]
Please create new regex for the core following the standard:

https://tools.ietf.org/html/rfc3696
